docs/autogen/templates/getting_started.rst.jinja

```diff
@@ -0,0 +1,726 @@
+  liburing etc.
+* ``libxnvme-static.a``, this is the same as ``libxnvme.a`` except it does
+  **not** come with **batteries included**, so you have to manually link with
+  SPDK, liburing, etc.
+* ``libxnvme-shared.so``, this is the shared library version of **xNVMe**, this
+  also does **not** come with **batteries included**, so when linking with, or
+  dynamically loading, the shared library version of **xNVMe** then you also
+  have to link or load the third-party dependencies.
+
+Using ``libxnvme.a`` is the preferred way to consume **xNVMe** as it comes with
+the correct version of the various third-party libraries and provides for
+a simpler link-target.
+
+Run!
+----
+
+.. literalinclude:: hello_01.cmd
+   :language: bash
+
+.. literalinclude:: hello_01.out
+   :language: bash
+
+This should conclude the getting started guide of **xNVMe**, go ahead and
+explore the :ref:`sec-tools`, :ref:`sec-c-api`, and :ref:`sec-examples`.
+
+Should **xNVMe** or your system still be misbehaving, then take a look in the
+:ref:`sec-troubleshooting` section or reach out by raising an issue_, start an
+asynchronous discussion_, or go to Discord_ for synchronous interaction.
+
+.. _sec-troubleshooting:
+
+Troubleshooting
+===============
+
+User space
+----------
+
+In case you are having issues using running with SPDK backend / using then make
+sure you following the config section
+:ref:`sec-gs-system-config-userspace-config` and if issues persist a solution
+might be found in the following subsections.
+
+No devices found
+~~~~~~~~~~~~~~~~
+
+When running ``xnvme enum`` then the output-listing is empty, there are no
+devices. When running with ``vfio-pci`` then this can occur when your devices
+are sharing iommu-group with other devices which are still bound to in-kernel
+drivers. This could be NICs, GPUs or other kinds of peripherals.
+
+The division of devices into groups is not something that can be easily
+switched, but you try to manually unbind the other devices in the iommu group
+from their kernel drivers.
+
+If that is not an option then you can try to re-organize your physical
+connectivity of deviecs, e.g. move devices around.
+
+Lastly you can try using ``uio_pci_generic`` instead, this can most easily be
+done by disabling iommu by adding the kernel option: ``iommu=off`` to the
+kernel command-line and rebooting.
+
+Memory Issues
+~~~~~~~~~~~~~
+
+If you see a message similar to the below while unbind devices::
+
+  Current user memlock limit: 16 MB
+
+  This is the maximum amount of memory you will be
+  able to use with DPDK and VFIO if run as current user.
+  To change this, please adjust limits.conf memlock limit for current user.
+
+  ## WARNING: memlock limit is less than 64MB
+  ## DPDK with VFIO may not be able to initialize if run as current user.
+
+Then go you should do as suggested, that is, adjust ``limits.conf``, see for an
+example of doing here :ref:`sec-gs-system-config-userspace-config`.
```

Comment:
see for an example of doing here -> for an example, see

Reply:
fixed